### ISSUE TITLE
Implement Accessibility Contrast Checker

### ISSUE THREAD

**Issue by suman20041** [YadavAkhileshh/OpenPlayground#561]
## Description
The current Color Palette Generator lacks accessibility features, which are essential for inclusive design. We need to add functionality to evaluate color combinations against WCAG (Web Content Accessibility Guidelines) standards .

## Implement

- Suggest accessible alternatives
- Export palette as CSS variables

## Screenshot

<img width="1893" height="829" alt="Image" src="https://github.com/user-attachments/assets/61dfd068-4f54-434f-9a48-9707c1cd85f3" />
